src/router/index.ts

```diff
@@ -11,6 +12,11 @@ const routes: Array<RouteRecordRaw> = [
     path: '/home',
     name: 'Home',
     component: Home
+  },
+  {
+    path: '/login',
+    name:'Login',
```

Comment:
please create an enum for the route names.

Reply:
Enum file created.